modules/helper.py

```diff
@@ -404,14 +404,13 @@ def autoRotate(ifile):
             result = debug.subprocess_check_output(['/usr/bin/jpegexiforient', ifile])  # , stderr=void)
         if result:
             orient = int(result)-1
-            if orient < 0 or orient >= len(parameters):
-                logging.info('Orientation was %d, not transforming it', orient)
+            if not 0 < orient < len(parameters):
```

Comment:
I didn't know this was legal to write, but very neat. I would have liked a parenthesis around what `not` is inverting, just to make it clear.

Reply:
Fine to add parens - makes it easier to read as well..    I changed it b/c the if never happened.  The logic for orient = 0 changes from the old to the new, and I just got carried away.